File: src/main/java/Pages/IndexPage.java
package Pages;



import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;



public class IndexPage extends BasePage{

    By btnLaptops = By.xpath("//a[contains(text(),'Sony vaio i5')]");
    //div[@class='list-group']/child::a[contains(text(),'Laptops')]

    WebDriver driver;


  public IndexPage(WebDriver driver) {
      this.driver = driver;

  }


    public void EnterToLaptopSection(){
        BasePage.WaitElement(btnLaptops);
        driver.findElement(btnLaptops).click();

    }


}

File: test/pageobjects/index.page.js


const Page = require('./page');


class indexPage extends Page {
  

    get btnLaptops () {
        return $("//a[contains(text(),'Laptops')]");
    }


    clicktoLaptop(){
       super.clicktoBtn(this.btnLaptops);
    }

   
    openDemozableSite() {
        return super.open('/');
    }
}

module.exports = new indexPage();
